### PR TITLE
added some kubernetes specific to roadmap

### PR DESCRIPTION
**Pull request by andycmaj**

### CODE REVIEW

File: roadmap/brainstorm.md [hunk 1]
```diff
@@ -9,26 +9,41 @@ Let's identify some themes on where platform and developer experience tooling ca
...
+- Ideal infrastructure-as-code experience would include
+  - Applications defined as much as possible in kubernetes manifest yaml templates
+  - Vanilla AWS dependencies (for things like RDS, Elasticache, Dynamo, Lambda, etc.) defined in terraform
+  - Terraform somehow driving kubernetes (native manifest templating and application) AND AWS resources
+  - Environment variables and secrets pulled from Global Config source-of-truth to render kubernetes templates via TF vars?
+  - Potentially a layer of abstraction around versioned Releases and Environments
+
+- Possible lines of investigation
+  - Helm Provider for Terraform
+  - Layer0 Environment/Release abstractions
```
Comment (diemonster) [inline roadmap/brainstorm.md]: Let's flesh out these abstractions (release & environment) a little more in the doc lest we forget. I liked what you mentioned re: releases being "a grouping of resources by a variable or few ie. version/release/commit/etc" and environment being "a grouping of resources that get deployed/updated together".

File: roadmap/brainstorm.md [hunk 2]
```diff
@@ -9,26 +9,41 @@ Let's identify some themes on where platform and developer experience tooling ca
...
+
+- Ideal infrastructure-as-code experience would include
+  - Applications defined as much as possible in kubernetes manifest yaml templates
+  - Vanilla AWS dependencies (for things like RDS, Elasticache, Dynamo, Lambda, etc.) defined in terraform
+  - Terraform somehow driving kubernetes (native manifest templating and application) AND AWS resources
+  - Environment variables and secrets pulled from Global Config source-of-truth to render kubernetes templates via TF vars?
+  - Potentially a layer of abstraction around versioned Releases and Environments
+
+- Possible lines of investigation
+  - Helm Provider for Terraform
```
Comment (diemonster) [inline roadmap/brainstorm.md]: Looks like this exists: https://github.com/mcuadros/terraform-provider-helm

File: roadmap/brainstorm.md (hunk 1 above)
Comment (andycmaj) [reply roadmap/brainstorm.md]: will do

File: roadmap/brainstorm.md (hunk 2 above)
Comment (andycmaj) [reply roadmap/brainstorm.md]: yep. just calling out as an example. i'll add link to some other inspirations

File: roadmap/brainstorm.md [hunk 3]
```diff
@@ -9,26 +9,73 @@ Let's identify some themes on where platform and developer experience tooling ca
...
+- Terraform integration
+  - native kubernetes manifests
+  - manifest templating (a-la Helm)
+  - Integration with Global Configuration as a data-source for manifest templates
+
+- Ideal infrastructure-as-code experience would include
+  - Applications defined as much as possible in kubernetes manifest yaml templates
+  - Vanilla AWS dependencies (for things like RDS, Elasticache, Dynamo, Lambda, etc.) defined in terraform
+  - Terraform somehow driving kubernetes (native manifest templating and application) AND AWS resources
+  - Environment variables and secrets pulled from Global Config source-of-truth to render kubernetes templates via TF vars?
```
Comment (zpatrick) [inline roadmap/brainstorm.md]: TF vars -> TF data sources pulling from "Global Config source-of-truth"

Comment (andycmaj) [reply roadmap/brainstorm.md]: yes that's better. will update

**Pull request merged**